doc/source/whatsnew/v2.0.2.rst

```diff
@@ -31,6 +31,7 @@ Bug fixes
 - Bug in :func:`api.interchange.from_dataframe` was unnecessarily raising on bitmasks (:issue:`49888`)
 - Bug in :func:`merge` when merging on datetime columns on different resolutions (:issue:`53200`)
 - Bug in :func:`read_csv` raising ``OverflowError`` for ``engine="pyarrow"`` and ``parse_dates`` set (:issue:`53295`)
+- Bug in :func:`read_csv` when defining ``dtype`` with ``bool[pyarrow]`` (:issue:`53390`)
```

Comment:
This is only for "c"/"python" parsers, right?

Might want to clarify that here.

Reply:
Yeah correct, updated.